### PR TITLE
Remove invalid comments in unit tests

### PR DESCRIPTION
**Pull request by CaoShuFeng**
**Release note**:

```NONE
```

### CODE REVIEW

**Comment by dims** [comment]
@k8s-bot ok to test

File: pkg/kubectl/cmd/attach_test.go [hunk 1]
```diff
@@ -227,7 +227,6 @@ func TestAttach(t *testing.T) {
 					body := objBody(codec, test.pod)
 					return &http.Response{StatusCode: 200, Header: defaultHeader(), Body: body}, nil
 				default:
-					// Ensures no GET is performed when deleting by name
```
Comment (zhangxiaoyu-zidif) [inline pkg/kubectl/cmd/attach_test.go]: It's better to update a proper comment. =)

Comment (CaoShuFeng) [inline pkg/kubectl/cmd/attach_test.go]: I think the error message "unexpected request" says what happened here.
:)

**Pull request closed without merging**